src/mixins/embed.js

```diff
@@ -105,13 +108,43 @@ export default {
     });
     const metadata = data && data.data.records[0].data.contentMetadata;
     const stakeholders = data && data.data.records[0].data.stakeholders;
+
+    const stakeholdersFirstId = (stakeholders[0] && stakeholders[0].entity && stakeholders[0].entity['@id']) || '';
+    let inputAddress = stakeholdersFirstId;
+    const addressLengthWithoutPrefixAnd1 = 38;
+    if (stakeholdersFirstId.startsWith('did:like:')) {
+      inputAddress = `like1${stakeholdersFirstId.slice(stakeholdersFirstId.length - addressLengthWithoutPrefixAnd1)}`;
+    } else if (inputAddress.startsWith('did:cosmos:')) {
+      inputAddress = `cosmos1${stakeholdersFirstId.slice(stakeholdersFirstId.length - addressLengthWithoutPrefixAnd1)}`;
+    }
+    let stakeholdersValidlikeWallet;
+    if (isValidAddress(inputAddress)) {
+      stakeholdersValidlikeWallet = changeAddressPrefix(inputAddress, 'like');
+    }
+    let address = '';
+    let stakeholdersName;
+    if (stakeholdersValidlikeWallet
+      || (stakeholders && stakeholders[0] && stakeholders[0].entity.name)) {
+      address = stakeholdersValidlikeWallet;
+      stakeholdersName = (stakeholders && stakeholders[0] && stakeholders[0].entity.name);
+    } else {
+      address = data && data.data && data.data.owner;
+    }
+    const likerData = await apiGetLikerDataByAddress(address)
```

Comment:
The above `if else` statement contains a change where `address` is undefined, so better refactor above, otherwise we would need to check `address` exists here.

Reply:
But if it doesn't exist, I can go to get data. If I can't get it, it will be regarded as no liker. For consistency, I will remove the previous `=''`

Reply:
i think if we know it has no wallet, we dont want to call an api to slow down the ui display by calling restful
but in this case we must have owner wallet